Document truly transactional emails v2 feature (#723)
mautic/user-documentation: docs/campaigns/campaign_builder.rst

     - Removes the Contact from the Do Not Contact - DNC - list.
   * - **Send a Webhook** 
     - Sends a Webhook to a defined URL, using the GET, POST, PUT, PATCH, or DELETE methods. Headers and data is customizable, and support the use of tokens, such as Contact fields and the Contact's IP address. For example, ``{contactfield=firstname}``
   * - **Send Email**
     - Sends an Email to the selected Contact. The action includes a **Repeatable email** toggle:

        * Set to **Yes** to allow sending the Email to the same Contact multiple times within the Campaign.
        * Set to **No** to send the Email only once across all sources. If a Contact has already received this non-repeatable Email, Mautic skips the send and moves them to the next Campaign step.

>         If you enabled **Send to unsubscribed contacts** for the selected Email, Mautic displays this information in the action configuration.

Reviewer: ```suggestion
      If you enabled **Send to unsubscribed contacts** for the selected Email, Mautic displays this information in the action configuration.
```

     - Removes the Contact from the Do Not Contact - DNC - list.
   * - **Send a Webhook** 
     - Sends a Webhook to a defined URL, using the GET, POST, PUT, PATCH, or DELETE methods. Headers and data is customizable, and support the use of tokens, such as Contact fields and the Contact's IP address. For example, ``{contactfield=firstname}``
   * - **Send Email**

>      - Sends an Email to the selected Contact. The action includes a **Repeatable email** toggle: set to **Yes** to allow sending the Email to the same Contact multiple times within the Campaign; set to **No** to send the Email only once across all sources. If the Contact has already received a non-repeatable Email from another source or the current Campaign, they aren't sent the Email again and the Contact progresses through the Campaign. If the selected Email has **Send to unsubscribed contacts** enabled, Mautic displays this information in the action configuration.

Reviewer: This is a bit hard to read on UI because it's too long. Not sure if this structure and indentations work in RST table, but we might want to restructure this for easy read. Also, use active instead of passive voice.

```suggestion
     - Sends an Email to the selected Contact. The action includes a **Repeatable email** toggle:
      
        * Set to **Yes** to allow sending the Email to the same Contact multiple times within the Campaign.
        * Set to **No** to send the Email only once across all sources. If a Contact has already received this non-repeatable Email, Mautic skips the send and moves them to the next Campaign step.
 
        If you enabled **Send to unsubscribed contacts** for the selected Email, Mautic displays this information in the action configuration.
```
